cam2imageGUI.py

```diff
@@ -88,7 +90,7 @@ def MissingInput():
 
 def chooseFiles(fileTypes=None, msg=messages["msgLoadingSVGs"], multiple=True):

```

Comment:
Add types and docstring + naming
def choose_files(file_types: str = None, msg: str = "", multiple: bool = True) -> bool: